awsomezhang/RestEasy-develop
# stripe setup

To test the code:

insert the key in line 4 of server.js
run both the react-app and the node server
go through the questions for the 'create a memorial' and fill out payment info
log in to stripe.com with the RE email and password 
you should now be able to view the stats of the payment you just made

ALSO (i meant to make another branch for this whoops)
but the signup button is now on the navbar
